eqt/ui/UIFormWidget.py

```diff
@@ -488,10 +471,10 @@ def __init__(self, parent=None):
 
 
 class FormDockWidget(QtWidgets.QDockWidget):
-    def __init__(self, parent=None, title=None):
+    def __init__(self, parent=None, title=''):
```

Comment:
why this default and accept `title=None`?

Reply:
honestly I'd prefer removing the lines below:

```python
        if title is None:
            title = ''
```

but I left them in just for backwards compatibility.

Reply:
> honestly I'd prefer removing the lines below:
> 
> ```python
>         if title is None:
>             title = ''
> ```
> 
> but I left them in just for backwards compatibility.

we added these lines in https://github.com/TomographicImaging/eqt/pull/102/files
to fix some problems. the default title = None was already set then.